packages/webpack-cli/lib/utils/cli-executer.js

```diff
@@ -49,16 +48,17 @@ async function prompter() {
     }
 
     return [...args, ...boolArgs];
-}
+};
 
-async function run() {
+const run = async () => {
     try {
         const args = await prompter();
         logger.info('\nExecuting CLI\n');
-        await runCLI(args);
+        return args;
     } catch (err) {
         logger.error(`Action Interrupted, use ${cyan('webpack-cli help')} to see possible options.`);
+        process.exit(2);
```

Comment:
@webpack/cli-team we need tests this, let's do it in separate PR, because we need do release today